### DISCUSSION TITLE
Error loading modules: Error opening directory “/usr/lib/x86_64-linux-gnu/udisks2/modules”: No such file or directory

### DISCUSSION THREAD

**Discussion by hxt21** [telekom-security/tpotce#1076] [Q&A]
In log the is this error 

Error loading modules: Error opening directory “/usr/lib/x86_64-linux-gnu/udisks2/modules”: No such file or directory

CODE_FILE | udisksmodulemanager.c:226
-- | --
CODE_FUNC | get_modules_list
GLIB_DOMAIN | udisks
PRIORITY | 4
THREAD_ID | 344
_BOOT_ID | 00961c77e0b34fdd978db6d94905ca23
_CAP_EFFECTIVE | 1ffffffffff
_CMDLINE | /usr/libexec/udisks2/udisksd
_COMM | udisksd
_EXE | /usr/libexec/udisks2/udisksd
_GID | 0
_HOSTNAME | rottenpiety
_MACHINE_ID | fe9557f2bfa44e92ae7f471665ef1041
_PID | 344
_SELINUX_CONTEXT | unconfined
_SOURCE_REALTIME_TIMESTAMP | 1650308345948640
_SYSTEMD_CGROUP | /system.slice/udisks2.service
_SYSTEMD_INVOCATION_ID | af8f2a67c24441eaa6c8d74b70cd2fcd
_SYSTEMD_SLICE | system.slice
_SYSTEMD_UNIT | udisks2.service
_TRANSPORT | journal
_UID | 0
__CURSOR | s=bfa9ae271ea34a1d8a4ffd3d62171a34;i=588f5;b=00961c77e0b34fdd978db6d94905ca23;m=1dcddce3;t=5dcf25abae9f7;x=ee9c76521dce7432
__MONOTONIC_TIMESTAMP | 500030691
__REALTIME_TIMESTAMP | 1650308345948663

</div>
</body>
</html>

**Comment by Ganesh-403** [comment]
@hxt21 
This log entry by itself is usually **not an indication of a serious problem**.

From what I could find, `udisksd` attempts to load optional modules from:

```text
/usr/lib/x86_64-linux-gnu/udisks2/modules
```

However, that directory is **only created when one or more optional `udisks2-*` modules (such as `udisks2-btrfs`) are installed**. On systems without any optional modules, the directory simply doesn't exist, yet `udisksd` still logs the message. This behavior has been discussed in Debian and FreedomBox communities and is generally considered cosmetic rather than a functional failure. :contentReference[oaicite:0]{index=0}

So the first thing I'd ask is:

- Are you experiencing any actual storage or disk management problems (mounting USB drives, partitions, etc.), or is this only a journal log entry?

If everything is working normally, this message can generally be ignored.

If you want to suppress the warning, you have two options:

1. Install an optional `udisks2-*` module (if you actually need one), which creates the `modules` directory.
2. If you simply want to eliminate the log message, creating the directory manually has been reported to stop the warning:

```bash
sudo mkdir -p /usr/lib/x86_64-linux-gnu/udisks2/modules
```

That said, I would only do this if the log message is bothering you—it doesn't address an underlying functional issue because there usually isn't one. :contentReference[oaicite:1]{index=1}

If you're seeing additional `udisks2` errors (for example, failures to initialize specific modules or problems mounting disks), those would be worth investigating separately, as they may indicate missing packages or configuration issues rather than the missing `modules` directory itself. :contentReference[oaicite:2]{index=2}

If this solves your problem, feel free to mark it as the accepted answer so others can find it easily.
